.changeset/five-berries-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@coinbase/onchainkit": minor
```

Comment:
Why `minor`? is this causing a breaking changes?

Reply:
it will be a breaking change for users of `V2 API` - but it is under `experimental` flag anyways so developers probably understand there will be breaking changes 

i can change this to `patch` if you think it's more suitable

Reply:
let's do `patch` as it is experimental.

Reply:
fixed